Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
 				wrap([$class: 'Xvnc', useXauthority: true]) {
 					sh """
 					mvn clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
-						-Pbuild-individual-bundles -Pbree-libs -Papi-check\
+						-Pbuild-individual-bundles Ptest-on-javase-19 -Pbree-libs -Papi-check\
```

Comment:
should be `-Ptest-on-javase-19` - is missing

Reply:
Thanks